.gitignore

```diff
@@ -17,3 +17,6 @@
 
 # Ignore uploaded files
 /public/uploads
+
+# Ignote simplecov coverage
+/coverage
```

Comment:
Are you using Sublime? We commonly have this problem with it. Git (and some people) get very upset with the lack of trailing newlines, but I think there's a way to configure it. You have an extension for that, right, @timkurvers?